### PR TITLE
[TASK] add first documentation setup

### PR DESCRIPTION
**Pull request by jaapio**
Following the default setup of other documentation projects this patch adss a first set of documentation to help future developer to maintain and improve the project.

More documentation is needed. But we should also link to the guides documentation. That needs to be written and published.

### CODE REVIEW

File: Documentation/Introduction/Index.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+..  include:: ../Includes.rst.txt
+
+..  _introduction:
+
+============
+Introduction
+============
+
+This is the documentation rendering tool for Typo3 projects. It is based on
+`phpDocumentor/guides`_ and can be used as a drop-in replacement for sphinx.
```
Comment (brotkrueml) [inline Documentation/Introduction/Index.rst:10]:
```suggestion
`phpDocumentor/guides`_ and can be used as a drop-in replacement for Sphinx.
```

File: Documentation/Introduction/Index.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+The tool is used by the auotmated documentation rendering system of the
```
Comment (brotkrueml) [inline Documentation/Introduction/Index.rst:11]:
```suggestion
The tool is used by the automated documentation rendering system of the
```

File: Documentation/KnownProblems/Index.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+..  _known-problems:
+
+==============
+Known problems
+==============
+
+Use this section for informing about any type of of problem.
+
+This extension does not offer anything ... and it is not in the scope of intended usage..
```
Comment (brotkrueml) [inline Documentation/KnownProblems/Index.rst:11]: I don't understand this sentence.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/KnownProblems/Index.rst (hunk 3 above)
Comment (jaapio) [reply Documentation/KnownProblems/Index.rst]: To much copy paste, removed it.

File: Documentation/Developer/Theme-customization.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Theme Customization
+===================
+
+The theme provided by this package is prepared to be integrated into the typo3
+website. Templates are written in twig_ and uses components provided by
+`phpDocumentor/guides`_. The theme is using bootstrap_ as a base framework.
+
+To customize the components provided by `phpDocumentor/guides`_ you have to follow
+the directory structure of the templates provided by this package. Every template
+can be overwritten by creating a new template with the same name in the same
```
[the remark this answers is not in the document]
Comment (jaapio) [reply Documentation/Developer/Theme-customization.rst:17]: I tried to rephrase this page a bit with extra information. I hope this makes it more clear now.

File: Documentation/Index.rst [hunk 5]
```diff
@@ -1,69 +1,55 @@
...
 
-=============
-render-guides
-=============
+.. _start:
 
-Mono-repo setup
-===============
+=========
+<project>
```
Comment (brotkrueml) [inline Documentation/Index.rst:6]: We should substitute this placeholder, although it is a duplicate of the package name.

```suggestion
render-guides
```

File: Documentation/Index.rst [hunk 6]
```diff
@@ -1,69 +1,55 @@
...
+    |release|
 
-If you add a new dependency to a package, you can run
+:Language:
+    en
 
-::
-    composer run monorepo:merge
+:Author:
+    TYPO3 contributors
```
Comment (brotkrueml) [inline Documentation/Index.rst:19]: Maybe "TYPO3 documentation contributors"? What do you think, @linawolf ?

Comment (linawolf) [reply Documentation/Index.rst:19]: I think we can leave it like that

File: Documentation/Installation/Index.rst [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+..  _installation:
+
+=============
+Installation
+=============
+
+This project is not a TYPO3 extension, but a standalone application
+used to render documentation. If you want to learn more about how to write
+documentation, please check the TYPO3 documentation website.
```
Comment (linawolf) [inline Documentation/Installation/Index.rst:11]:
```suggestion
documentation, please check the :ref:`Contribtuib Guide - How to Document <2document:contribute>`
```

You also need to add "h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/"  in the guides.xml to the interlink mapping

Comment (garvinhicking) [reply Documentation/Installation/Index.rst:11]: (Typo in "Contribtuib" -> "Contributing")

**Review by linawolf — APPROVED** [review]

**Pull request merged**